Assets/Scripts/Model/Core.cs

```diff
@@ -84,6 +86,18 @@ internal void AddPlayer(Player player)
             }
         }
 
+        internal void AddPosition(string id, Position position)
```

Comment:
This should not have a `string id` parameter. Like the player, this id should be fetched from the position object.